pkg/gcp/client.go

```diff
@@ -233,3 +265,14 @@ func (c *gcpClient) UndeleteWorkloadIdentityPool(ctx context.Context, resource s
 func (c *gcpClient) UpdateRole(ctx context.Context, request *adminpb.UpdateRoleRequest) (*adminpb.Role, error) {
 	return c.iamClient.UpdateRole(ctx, request)
 }
+
+func (c *gcpClient) UpdateWorkloadIdentityPoolOidcIdentityProvider(
+	ctx context.Context,
+	provider *iamv1.WorkloadIdentityPoolProvider,
+) error {
+	_, err := c.oldIamClient.Projects.Locations.WorkloadIdentityPools.Providers.Patch(
+		provider.Name,
+		provider,
+	).UpdateMask("attributeMapping,description,displayName,disabled,state,oidc").Do()
+	return err
```

Comment:
does this err not have to be handled same as other error like at @ line#195

Reply:
ack.